src/Microsoft.SymbolManifestGenerator/Microsoft.SymbolManifestGenerator.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
You need to add this csproj to diagnostics.sln in order for it to be built and packaged as part of CI/Official builds.